frontend/lib/routes/company/items/AddItemForm.dart

```diff
@@ -0,0 +1,247 @@
+              content: Text('An error occured.',
+                  style: TextStyle(color: Colors.white))),
+        );
+      }
+      Navigator.pop(context);
+    }
+  }
+
+  Widget _buildForm() {
+    return Form(
+      key: _formKey,
+      child: Column(
+        children: [
+          Padding(
+            padding: const EdgeInsets.all(8.0),
+            child: TextFormField(
+              controller: _nameController,
+              validator: (value) {
+                if (value == null || value.isEmpty) {
+                  return 'Please enter a name';
+                }
+                return null;
+              },
+              decoration: const InputDecoration(
+                icon: const Icon(Icons.title),
+                labelText: "Name *",
+              ),
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.all(8.0),
+            child: TextFormField(
+              controller: _descriptionController,
+              validator: (value) {
+                if (value == null || value.isEmpty) {
+                  return 'Please enter a description';
+                }
+                return null;
+              },
+              decoration: const InputDecoration(
+                icon: const Icon(Icons.title),
+                labelText: "Description *",
+              ),
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.all(8.0),
+            child: TextFormField(
+              controller: _typeController,
+              validator: (value) {
+                if (value == null || value.isEmpty) {
+                  return 'Please enter a type';
+                }
+                return null;
+              },
+              decoration: const InputDecoration(
+                icon: const Icon(Icons.title),
+                labelText:
+                    "Type * (e.g Publicity, Merchandise, Stands, Talk or other)",
+              ),
+            ),
+          ),
+          Row(
+            children: [
+              Expanded(
+                child: Padding(
+                  padding: const EdgeInsets.all(8.0),
+                  child: TextFormField(
+                    controller: _valueEurosController,
+                    validator: (value) {
+                      if (value == null || value.isEmpty) {
+                        return 'Please enter the cost of the item';
+                      }
+                      return null;
+                    },
+                    decoration: const InputDecoration(
+                      icon: const Icon(Icons.money),
+                      labelText: "Cost of item (only euros) *",
+                    ),
+                    keyboardType: TextInputType.number,
+                    inputFormatters: <TextInputFormatter>[
+                      FilteringTextInputFormatter.digitsOnly
+                    ],
+                  ),
+                ),
+              ),
+              Expanded(
```

Comment:
Maybe we should have a field where people can insert any amount (e.g. 12.50) and then convert it internally to only cents for example?